pkg/app/pagewriter/error.html

```diff
@@ -59,6 +59,9 @@ <h1 class="subtitle is-1">{{.Title}}</h1>
         <div class="content">
           {{.Message}}
         </div>
+        <div class="content">
```

Comment:
Is this where (and how) we want the Request ID included in the error page?

Reply:
Potentially add Or around if `.Message` or `.RequestId` is set (`.Message` could be blank)

Reply:
Yep, I think this looks good :slightly_smiling_face: